api/v1/controllers/RegistrationController.js

```diff
@@ -1,17 +1,45 @@
+var _ = require('lodash');
 var bodyParser = require('body-parser');
+var _Promise = require('bluebird');
 
 var services = require('../services');
 var middleware = require('../middleware');
 var requests = require('../requests');
 var roles = require('../utils/roles');
 var mail = require('../utils/mail');
+var registration = require('../utils/registration');
+var errors = require('../errors');
 
 var router = require('express').Router();
 
 function _isAuthenticated (req) {
 	return req.auth && (req.user !== undefined);
 }
 
+function _validateGetAttendeesRequest(page, count, category, ascending){
```

Comment:
I think it would make more sense to have an AttendeeReview model or something like that and then create a hasOne or hasMany relation on Attendee, allowing you to write a .prototype.validate for an AttendeeReviewRequest. I'm not sure which is better at the end so it's a question for @nmagerko imo.